platform/barefoot/sonic-platform-modules-ingrasys/s9180-32x/utils/i2c_utils.sh

```diff
@@ -174,12 +174,18 @@ function _i2c_init {
     echo "# Description: I2C Init"
     echo "========================================================="
 
+    depmod -a
```

Comment:
bad practice.

Reply:
if I don't add this, some kernel modules will modprobe fail in first boot after sonic installation. Although the "depmod -a" already add in platform/barefoot/sonic-platform-modules-ingrasys/debian/sonic-platform-ingrasys-s9180-32x.init, but seem not work in stretch build. Any suggestion for better solution ?

Reply:
check here. https://github.com/Azure/sonic-buildimage/blob/master/platform/broadcom/sonic-platform-modules-dell/debian/platform-modules-z9264f.postinst